### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#156296]
Saw this in dev tools after restart with an editor opened that had sticky scroll:

```
TypeError: Cannot read properties of undefined (reading 'length')
    at Function.filter (lineDecorations.ts:64:23)
    at i.getDomNode (stickyScroll.ts:235:45)
    at S.updateRootNode (stickyScroll.ts:330:38)
    at r._renderStickyScroll (stickyScroll.ts:212:27)
    at r._update (stickyScroll.ts:78:8)
```

**Comment by aiday-mar** [comment]
The code block creating this issue is now in a Try Catch statement, this is the output of the console log in the catch statement. Later I'll look more closely at the file lineDecorations.ts, from which the error is originating.

**Comment by aiday-mar** [comment]
Fixed with 95fd5bf198bc9f0083421ce5a53c05ca0294a7fd
